ibis/expr/rewrites.py

```diff
@@ -211,7 +211,7 @@ def empty_in_values_right_side(_):
 
 @replace(
     p.WindowFunction(
-        p.PercentRank(y) | p.RankBase(y) | p.CumeDist(y) | p.NTile(y),
```

Comment:
None of `PercentRank`, `RankBase` or `CumeDist` has an argument.

Reply:
Good catch!